src/solvers/smt2/smt2_conv.cpp

```diff
@@ -53,7 +53,8 @@ smt2_convt::smt2_convt(
   const std::string &_logic,
   solvert _solver,
   std::ostream &_out)
-  : use_FPA_theory(false),
+  : use_as_const(false),
+    use_FPA_theory(false),
```

Comment:
Nit picking: in the interest of reducing churn, could you swap those two lines (and then also the declarations in the header file)?

Reply:
Sure, I'll make this change.

Thanks for taking a look!